elyra/metadata/schemas/kfp.json

```diff
@@ -21,33 +21,51 @@
       "properties": {
         "description": {
           "description": "The description of this KFP runtime instance",
-          "type": "string"
+          "type": "string",
+          "uihints": {
+            "label": "Description"
+          }
```

Comment:
Please note that I just created #781 to use a `title` field to convey the display_name/label for each property.  This would allow the removal of the `uihints` stanzas until there are actual needs for edit-controls or other UI-specific behaviors should it be merged.  As a result, we might want to look at that PR soon and remove these updates if merged.

Reply:
I'll review that asap, I agree that it makes sense for it to go in first, plus this is has a final todo on it anyway